distribution/helm/vector-agent/templates/configmap.yaml

```diff
@@ -6,11 +8,44 @@ metadata:
   labels:
     {{- include "libvector.labels" . | nindent 4 }}
 data:
+  {{- if .Values.customConfig }}
+  {{- if and .Values.customConfig $deprecatedConfig }}{{ fail "Can't use deprecated keys with customConfig" }}{{- end }}
```

Comment:
Should this error be more descriptive, to indicate the used (or all) deprecated keys?

Reply:
I can't think of a reasonable way to identify which may have been used and selectively print off the top of my head, and the full list was just _long_.

I can think on the first and otherwise print the whole list.